index.html

```diff
@@ -32,6 +32,7 @@
     <script src="scripts/scripts.js"></script>
```

Comment:
Please update debig/indexs.html as well

Reply:
debug*

Reply:
I think we should update it after all changes are commited.

Reply:
Done